src/main/io/serial_cli.c

```diff
@@ -2295,6 +2307,31 @@ static void cliSet(char *cmdline)
                             }
                         }
                         break;
+                    case MODE_PIN: {
+                            uint8_t pin = 0;
+                            if (strlen(eqptr) > 0) {
+                                if (strcasecmp(eqptr, "NONE") == 0) {
+                                    tmp.int_value = 0;
+                                    changeValue = true;
+                                } else {
+                                    uint8_t port = (*eqptr)-'A';
+                                    if (port < 8) {
+                                        eqptr++;
+                                        pin = atoi(eqptr);
+                                        if (pin < 16) {
+                                            tmp.int_value = DEFIO_TAG_MAKE(port, pin);
+                                            ioRec_t *rec = IO_Rec(IOGetByTag(tmp.int_value)); 
+                                            if (rec && rec->owner == OWNER_FREE) {
```

Comment:
This will cause problems when user wants to swap pins - reset will be necessary ..